dangerfile.js

```diff
@@ -0,0 +1,26 @@
+import { fail, danger } from 'danger';
+import { flatten, intersection, isEmpty, includes } from 'lodash';
+
+const pkg = require('./package.json'); // eslint-disable-line import/newline-after-import
+const prLogConfig = pkg['pr-log'];
+
+const checkRequiredLabels = labels => {
+  const requiredLabels = flatten([
+    prLogConfig.skipLabels || [],
+    Object.keys(prLogConfig.validLabels || {}),
+  ]);
+
+  if (includes(labels, 'do not merge')) {
+    fail('PR is marked with "do not merge" label.');
+  }
+
+  const foundLabels = intersection(requiredLabels, labels);
+  if (isEmpty(foundLabels)) {
+    fail(`PR is not labeled with one of: ${JSON.stringify(requiredLabels)}`);
+  }
```

Comment:
This makes sense, but only if you consider that an org contributor can set labels - so an OSS contribution wouldn't be able to do this - so it might be worth changing it to warn ( or to check whether the PR author is in the storybooks org )